Description: Removed files from POTFILES.in because they are not found.
 Remove file names from POTFILES.in list because upstream forgot to
 add under source tree.

Forwarded: Yes
Author: Elías Alejandro Año Mendoza <ealmdz@gmail.com>
Last-Update: 2016-04-05

--- uget.orig/po/POTFILES.in
+++ uget/po/POTFILES.in
@@ -1,9 +1,7 @@
 uget/pwmd.c
 uget/UgetApp.c
 uget/UgetEvent.c
-uget/UgetMedia-youtube.c
 uget/UgetPluginAria2.c
-uget/UgetPluginMedia.c
 ui-gtk/UgtkAboutDialog.c
 ui-gtk/UgtkApp-main.c
 ui-gtk/UgtkApp-timeout.c
